packages/data-stores/src/launch/selectors.ts

```diff
@@ -84,5 +84,9 @@ export const isModalDismissible = ( state: State ): boolean => state.isModalDism
 // Check if launch modal title should be visible
 export const isModalTitleVisible = ( state: State ): boolean => state.isModalTitleVisible;
 
+// Check if launch Success view should be displayed (user didn't dismissed the Success View modal)
+export const shouldDisplaySuccessView = ( state: State ): boolean => state.shouldDisplaySuccessView;
+
 // Check if launch modal can be dismissed
-export const isFocusedLaunchOpen = ( state: State ): boolean => state.isFocusedLaunchOpen;
+export const isFocusedLaunchOpen = ( state: State ): boolean =>
+	state.isFocusedLaunchOpen || select( LAUNCH_STORE ).shouldDisplaySuccessView();
```

Comment:
We're definitely [overusing selectors](https://daveceddia.com/redux-selectors/#you-probably-dont-need-selectors-for-everything) here. All these are not computing derived data from the store.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/47808/commits/d5b58b653ecde65082a4d4f0f7115dd8ba89dd1a